src/main/java/com/datastax/cdm/cql/statement/TargetUpsertRunDetailsStatement.java

```diff
@@ -36,45 +38,66 @@ public class TargetUpsertRunDetailsStatement {
     private long prevRunId;
     private BoundStatement boundInitInfoStatement;
     private BoundStatement boundInitStatement;
-    private BoundStatement boundUpdateInfoStatement;
+    private BoundStatement boundEndInfoStatement;
     private BoundStatement boundUpdateStatement;
     private BoundStatement boundUpdateStartStatement;
+    private BoundStatement boundSelectInfoStatement;
     private BoundStatement boundSelectStatement;
 
     public TargetUpsertRunDetailsStatement(CqlSession session, String keyspaceTable) {
         this.session = session;
         String[] ksTab = keyspaceTable.split("\\.");
+        if (ksTab.length != 2) {
+            throw new RuntimeException("Invalid keyspace.table format: " + keyspaceTable);
+        }
         this.keyspaceName = ksTab[0];
         this.tableName = ksTab[1];
         String cdmKsTabInfo = this.keyspaceName + ".cdm_run_info";
         String cdmKsTabDetails = this.keyspaceName + ".cdm_run_details";
 
         this.session.execute("create table if not exists " + cdmKsTabInfo
                 + " (table_name text, run_id bigint, run_type text, prev_run_id bigint, start_time timestamp, end_time timestamp, run_info text, primary key (table_name, run_id))");
-        this.session.execute("create table if not exists " + cdmKsTabDetails
-                + " (table_name text, run_id bigint, start_time timestamp, token_min bigint, token_max bigint, status text, primary key ((table_name, run_id), token_min))");
+        try {
+            this.session.execute("alter table " + cdmKsTabInfo + " add status text");
+        } catch (Exception e) {
+            // ignore if column already exists
```

Comment:
do we just do an INFO level logging here?

Reply:
Fixed. Also added note to remove this whole try/catch block after a few releases.